packages/cms-lib/logger.js

```diff
@@ -96,6 +104,16 @@ const logger = {
       currentLogger.log(...args);
     }
   },
+  success(...args) {
+    if (shouldLog(LOG_LEVEL.LOG)) {
+      currentLogger.success(...args);
```

Comment:
I think we probably need to check for `currentLogger.success` since that is a bit non-standard and fall back to `currentLogger.log()`. For example, I am curious about whether or not this impacts the logging in the webpack plugin.